tests/queries/0_stateless/02969_mysql_cast_type_aliases.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
The new type aliases are motivated by MySQL but they are of general usefulness and not part of the MySQL handler in ClickHouse. As a result, the tests should be be normal SQL tests instead of MySQL tests.

Reply:
I see your point. I will rewrite them, and it's easier that way.